src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,66 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (n: number) => void
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  return (
+    <ul className="pagination">
+      <li className={cn('page-item', { disabled: currentPage === 1 })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
+          onClick={() => (
+            currentPage !== 1 && onPageChange(currentPage - 1)
+          )}
+        >
+          «
+        </a>
+      </li>
+      {getNumbers(1, totalPages).map(n => (
+        <li
+          key={n}
+          className={cn('page-item', { active: n === currentPage })}
+        >
+          <a
+            data-cy="pageLink"
+            className="page-link"
+            href={`#${n}`}
+            onClick={() => onPageChange(n)}
```

Comment:
Move to a separate function

Reply:
In this case if I create separate function I can't pass an argument from cycle's scope